ara/api/views.py

```diff
@@ -95,6 +95,25 @@ class HostViewSet(viewsets.ModelViewSet):
     queryset = models.Host.objects.all()
     filterset_class = filters.HostFilter
 
+    def perform_destroy(self, instance):
```

Comment:
There is a problem with this implementation: this is run when a host is deleted via the API but not when the host is deleted from a cascade reference -- for example, if I delete a playbook, it will delete the hosts referring to it but won't update the latesthost because this part doesn't run.

Reply:
Tested a fix by adding a similar perform_destroy in the playbook view: https://github.com/ansible-community/ara/pull/308